shipyard_blueprints/shipyard-bp-utils/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "shipyard-bp-utils"
-version = "0.1.1"
+version = "1.0.0a1"
```

Comment:
Major change- i feel like this would be appropriate with the major refactoring project coming up. Less concerned about backwards compatibility

Reply:
I wouldn't say we are not concerned with backwards compatibility, because the hope is that when we use this as a drop-in replacement for `shipyard-utils`, the functionality doesn't change for existing fleets/vessels using those functions